scaladoc/src/dotty/tools/scaladoc/renderers/MemberRenderer.scala

```diff
@@ -323,6 +323,7 @@ class MemberRenderer(signatureRenderer: SignatureRenderer)(using DocContext) ext
       val (allInherited, allDefined) = nonExperimental.partition(isInherited)
       val (depDefined, defined) = allDefined.partition(isDeprecated)
       val (depInherited, inherited) = allInherited.partition(isDeprecated)
+      val (abstractInherited, concreteInherited) = inherited.partition(isAbstract)
```

Comment:
Note:
If we just put inherited abstract members into  "Abstract members" section, it loses the information that these methods are originated from inheritance.